### ISSUE TITLE
job.attributes() missing attributes from ATTR_GROUP_JOB_TMPL

### ISSUE THREAD

**Issue by Mr-Magnificent** [watson/ipp-printer#34]
Is there any way to see the values of attributes of submitted job from **`ATTR_GROUP_JOB_TMPL`** array in **`/lib/utils.js`**? Need to find the orientation of the submitted job.

**Comment by yeganehkordi** [comment]
Did you find anything? I need to find this attributes.
@Mr-Magnificent

**Comment by Mr-Magnificent** [comment]
No, Didn't find anything but made a workaround hack with mapping dimensions of the page to orientation.
